File: class02/cpp/AP0201-AprovadoReprovado.cpp
#include <iostream>
int main() {
    double nota;
    std::cout << "Nota: "; std::cin >> nota;
    if (nota >= 7) std::cout << "Aprovado" << std::endl;
    else std::cout << "Reprovado" << std::endl;
    return 0;
}

File: class02/java/AP0201-AprovadoReprovado.java
import java.util.Scanner;
public class Aprovado {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.print("Nota: ");
        double nota = sc.nextDouble();
        if (nota >= 7) System.out.println("Aprovado");
        else System.out.println("Reprovado");
    }
}
